Prog/reshape ACS data for population projection.do

```diff
@@ -0,0 +1,568 @@
+********************************************************************************
+* Author: Yipeng Su
+* Project: Regional Housing Framework
+* Date: 12/28/2018
+* Moderation:
+********************************************************************************
+
+import excel "L:\Libraries\RegHsg\Raw\ACS 1 year estimate\All_1year.xlsx", sheet("Sheet1") firstrow
+
+label variable HC01_EST_VC01 "totpop"
```

Comment:
these labels should be written out especially as they mirror the vars created below. you can use the metadata files from the aff download to make this easier.